server/schedule/operator/operator.go

```diff
@@ -125,6 +125,18 @@ func (o *Operator) Kind() OpKind {
 	return o.kind
 }
 
+// SchedulerKind return the highest OpKind even if the operator has many OpKind
+// fix #3778
+func (o *Operator) SchedulerKind() OpKind {
```

Comment:
We can rearrange OpKind and use the LowBit function to get it directly.
`LowBit(x) = x&(-x)`

Reply:
Perhaps, we can find a better name.